test/extended/util/cluster/cluster.go

```diff
@@ -128,9 +137,10 @@ func LoadConfig(state *ClusterState) (*ClusterConfiguration, error) {
 	zones.Delete("")
 
 	config := &ClusterConfiguration{
-		MultiMaster: len(state.Masters.Items) > 1,
-		MultiZone:   zones.Len() > 1,
-		Zones:       zones.List(),
+		MultiMaster:           len(state.Masters.Items) > 1,
```

Comment:
hm... can MultiMaster be True and SingleReplica topology can be 1?

Reply:
"We're not really sure let's just create a new field" is the conclusion from [this](https://github.com/openshift/origin/pull/26314#discussion_r667475569) resolved conversation

Reply:
I think ``MultiMaster`` is not needed
Because the only usage in this repo refers to something which can be handled by ``MultiZone`` field imho

Reply:
ControlPlaneTopology == == configv1.SingleReplicaTopologyMode is true only for SNO.
So as long we don't add master nodes to SNO, when SingleReplicaTopology is true,  multiMaster is false and vice versa.

Reply:
> I think `MultiMaster` is not needed
> Because the only usage in this repo refers to something which can be handled by `MultiZone` field imho

This repo is also running vendored tests from the k8s repo, which originally defined the MultiMaster variable in the first place, so it makes sense to populate it just in case upstream makes use of it